components/input/InputBase.js

```diff
@@ -0,0 +1,191 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import omit from 'lodash.omit';
+import Box, { omitBoxProps, pickBoxProps } from '../box';
+import { ErrorText, HelpText } from '..';
```

Comment:
PP: Care to have it explicitly imported? Don't want to (potentially) lose hours again figuring out what's going wrong when `npm link`-ing